buildspecs/merge-build.yml

```diff
@@ -128,5 +147,13 @@ artifacts:
     # rpm
     - $ECS_AGENT_RPM
 
+    # csi driver
```

Comment:
do these artifacts need to be added to `signing.yml` script to get the pipeline working end to end?

Reply:
yes they do, i am working on that now